### ISSUE TITLE
Review tab is blank

### ISSUE THREAD

**Issue by darylknight** [dispositiontools/craft-alt-text-generator#14]
I assume that the first tab on the Alt text Generator (which maybe should be "Alt Text Generator", capital T) is meant to show some kind of overview, but the table is blank:

![image](https://github.com/user-attachments/assets/1b4ed0a7-67dc-4655-bfff-970413153906)

**Comment by dispositiontools** [comment]
Thank you for spotting the lower case t - yeah, I'll get that fixed in the next release!

The first tab should be any pending alt texts ready for being reviewed - but thats not actually clear. It should have a title of Review instead of Dashboard. 

I'm sorry. I'll get this sorted in the next release as well.

**Comment by dispositiontools** [comment]
i've pushed updates v4.2.1 and v5.2.1 which updates this first page to make it clear it is showing the items for review. 
I've also updated the composer.json to update the main nav item to Alt Text Generator - but currently that only works on newly installed. I'll add a migration another day to fix it on currently installed items. 

Currently working on updates for multi language.
